common/src/main/scala/explore/common/UserPreferencesQueries.scala

```diff
@@ -106,7 +106,8 @@ object UserPreferencesQueries {
       userId:       Option[User.Id],
       area:         ResizableSection,
       defaultValue: Int
-    )(implicit cl:  GraphQLClient[F, UserPreferencesDB]): F[Int] =
+    )(implicit cl:  GraphQLClient[F, UserPreferencesDB]
+    ): F[Int] =
```

Comment:
Ouch

Reply:
Yeah, this falls into the "some are a little worse" category. I do like the other places where it puts the final case class closing paren on the same line as the last parameter, though. 🤷